examples/Example3/Makefile

```diff
@@ -0,0 +1,90 @@
+# SPDX-FileCopyrightText: 2020 CERN
+# SPDX-License-Identifier: Apache-2.0
+
```

Comment:
That's how I started the code, as I didn't want to waste time creating a CMake build system just for a throw away example. I just didn't remove the Makefile afterwards because I still use that to build the code rather than CMake. It's easier to play around with compile flag changes, etc, using the Makefile than it is using CMake. More importantly, though, there are small differences in the results compiled with make and CMake that I want to try to understand before removing the Makefile. When the CUDA code is compiled as a whole instead of one object at a time, there are minor differences in the raytracing example/test, which are also influenced by what compile flags get used. Once those are well understood, the Makefile will become redundant and can be removed.